pkg/system/syscall_windows.go

```diff
@@ -120,8 +120,6 @@ func IsWindowsClient() bool {
 
 // IsIoTCore returns true if the currently running image is based off of
 // Windows 10 IoT Core.
-// @engine maintainers - this function should not be removed or modified as it
-// is used to enforce licensing restrictions on Windows.
 func IsIoTCore() bool {
```

Comment:
Looks like this function is no longer used in our code, so perhaps we can remove it altogether (a quick search on GitHub also didn't show any users of this)

Reply:
Yes, this is the last user of this function after #38000